pkg/Library/Graph/Atoms/GraphAgent.js

```diff
@@ -17,10 +17,7 @@ async update(inputs, state, {output, isDirty, service}) {
   }
   if (inputs.publishedGraphsUrl && isDirty('publishedGraphsUrl')) {
     state.publishedGraphsUrl = inputs.publishedGraphsUrl;
-    const publicGraphs = await this.loadPublicGraphs(inputs.publishedGraphsUrl);	
-    if (publicGraphs) {
-      output({publicGraphs});
-    }
+    output(await this.loadPublicGraphs(inputs.publishedGraphsUrl));
```

Comment:
try to avoid using `output` unless we need specifically to output something before going async again.

Reply:
same as above: I would prefer doing a separate refactoring PR, as GraphAgent is already somewhat fragile :(